pulp_container/app/json_schemas.py

```diff
@@ -1,19 +1,22 @@
 from pulp_container.constants import (
     ALLOWED_ARTIFACT_TYPES,
-    ALLOWED_BLOB_CONTENT_TYPES,
     BLOB_CONTENT_TYPE,
     MEDIA_TYPE,
     SIGNATURE_TYPE,
 )
 
 
 def get_descriptor_schema(
-    allowed_media_types, additional_properties=None, additional_required=None
+    allowed_media_types=[], additional_properties=None, additional_required=None
```

Comment:
Defining an empty list as a default argument in a function may pose side effects.
```
>>> def foo(param=[]):
...     param.append(1)
...     return param
... 
>>> 
>>> foo()
[1]
>>> foo()
[1, 1]
```